app/org/sagebionetworks/bridge/redis/JedisTransaction.java

```diff
@@ -37,44 +34,32 @@ public JedisTransaction del(final String key) {
     }
 
     public List<Object> exec() {
-        try (Jedis jedis = this.jedis) {
+        try {
             return transaction.exec();
+        } finally {
+            close();
```

Comment:
Isn't `close()` in finally block the same as `try (Jedis jedis = this.jedis)`?  It's just syntax sugar, right?